File: Programming/Solar/p1.java
import java.util.Scanner;

class p1 {
    public static void main(String[] args) {
        Scanner scan = new Scanner(System.in);

        // String name = scan.next(); // one word
        String full_name = scan.nextLine();

        System.out.println(full_name.charAt(0));

        // System.out.println(full_name.charAt(0));

        // int number = scan.nextInt(); // Integer
        // double number2 = scan.nextDouble(); // double
        // float number3 = scan.nextFloat(); // float

        scan.close();
        // System.out.println("Hello to you " + full_name);
    }
}

File: Assignment/a9/p1.c
#include <stdio.h>

int main() {
    char s[20]="Hello", t[20];
    int i,len=0;

    while(s[len]!='\0')
        len++;

    for(i=0;s[i]!='\0';i++)
        t[i]=s[i];
    t[i]='\0';

    printf("Length=%d\n%s",len,t);

    return 0;
}